memcached-operator/pkg/controller/memcached/memcached_controller.go

```diff
@@ -208,44 +196,70 @@ func (r *ReconcileMemcached) deploymentForMemcached(m *cachev1alpha1.Memcached)
 				},
 				Spec: corev1.PodSpec{
 					Containers: []corev1.Container{{
-						Image:   "memcached:1.4.36-alpine",
```

Comment:
Would not the direct way make more implicit for new users what are the values used for each attribute?